android/app/src/main/assets/capacitor.config.json

```diff
@@ -6,7 +6,16 @@
   "webDir": "build",
   "plugins": {
     "SplashScreen": {
-      "launchShowDuration": 0
+      "launchShowDuration": 10000,
+      "launchAutoHide": false,
```

Comment:
Hey @rizwan2000rm you can go for reducing the `launchShowDuration` to `1500` as `10,000` is not the ideal duration and make  `launchAutoHide` to `true` :)

Reply:
I think the launchShowDuration even matter if launchAutoHide is set to false. Just left there incase.

With this approach we show the splash screen for exactly the amount of seconds required to get to home screen. If it is 1s then splash screen takes 1s and if is 5s then splash screen takes 5s.

I think this is a better approach rather than just making the user wait for some fix amount of time.

Reply:
Okay, @rizwan2000rm  got the point. I suggested these changes because while making the same changes in my local environment the application is taking too much time to get into the home screen, it was exceeding the actual splash screen time duration that is around 2 to 3 seconds!

Reply:
I think you should confirm it by cloning my actual branch. I have also linked the video but still we should actually test it on more devices.

I didn't had much delay on my devices, it takes around 3 sec or less.

You can try it with your devices on my branch and let me know the results.

But the basic idea is, it would take longer than 3-4s if the device takes longer to load. If we remove the splash before we will get a blank white screen.

Reply:
> But the basic idea is, it would take longer than 3-4s if the device takes longer to load. If we remove the splash before we will get a blank white screen.

Yeah... I think it might be resolved when the app is built in release mode if that would be the case anyways I'll test your branch and let you know!